keyboards/zen/rev2/keymaps/default/keymap.c

```diff
@@ -0,0 +1,117 @@
+    KC_LCTL,          KC_LGUI, KC_LALT, RGB_TOG, MO(_NAV), KC_SPC,  KC_DEL,  KC_ENT,   KC_SPACE, MO(_NAV), KC_MINS, KC_EQL,  KC_PGUP, KC_PGDN, \
+                                                           KC_SPC,  KC_DEL,  KC_ENT,   KC_SPACE \
+  ),
+
+  /* NAV
+   * ,-----------------------------------------.              .-----------------------------------------.
+   * | GESC |   1  |   2  |   3  |   4  |   5  |              |   6  |   7  |   8  |   9  |   0  | Bksp |
+   * |------+------+------+------+------+------|              |------+------+------+------+------+------|
+   * | Tab  |   Q  |   W  |   E  |   R  |   T  |              |   Y  |   U  |   I  |   O  |   P  |   \  |
+   * |------+------+------+------+------+------|              |------+------+------+------+------+------|
+   * |CAPS(NAV)|A  |   S  |   D  |   F  |   G  |              |   H  |   J  |   K  |   L  |   ;  |  "   |
+   * |------+------+------+------+------+------+------..------+------+------+------+------+------+------|
+   * | Shift|   Z  |   X  |   C  |   V  |   B  |      ||      |   N  |   M  |   ,  |   .  |   /  |Enter |
+   * |------+------+------+------+------+------+------||------+------+------+------+------+------+------|
+   * | Ctrl | GUI  | Alt  |RGBTOG| NAV  |Space |Delete||Enter |Space |  NAV |   -  |   =  | PGUP | PGDN |
+   * `----------------------------------+-------------''------------------------------------------------'
+   *                                    |Space |Delete||Enter |Space |
+   *                                    '-------------''-------------'
+   */
+  [_NAV] = LAYOUT( \
+    KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,                     KC_F7,   KC_F8,    KC_F9,   KC_F10,  KC_F11,  KC_F12,  \
+    _______, RGB_SAI, RGB_VAI, RGB_SAD, RESET,   KC_LBRC,                   KC_RBRC, KC_PGUP,  KC_UP,   KC_PGDN, KC_INS,  KC_HOME, \
+    _______, RGB_HUD, RGB_VAD, RGB_HUI, _______, _______,                   _______, KC_LEFT,  KC_DOWN, KC_RGHT, KC_DEL,  KC_END,  \
+    KC_LSFT, _______, _______, _______, _______, _______, _______, _______, _______, _______,  _______, KC_MPLY, KC_MPRV, KC_MNXT, \
+    KC_LCTL, KC_LGUI, KC_LALT, RGB_MOD, _______, _______, _______, _______, _______, _______,  _______, KC_MUTE, KC_VOLU, KC_VOLD, \
+                                                 _______, _______, _______, _______
+  ),
+
+};
+
+void encoder_update_user(uint8_t index, bool clockwise) {
+  if (index == 0) { /* First encoder */
+    if (clockwise) {
+      tap_code(KC_PGDN);
+    } else {
+      tap_code(KC_PGUP);
+    }
+  } else if (index == 1) { /* Second encoder from slave */
+    if (clockwise) {
+      tap_code(KC_UP);
+    } else {
+      tap_code(KC_DOWN);
+    }
+  }
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
+        set_single_persistent_default_layer(1UL<<_QWERTY);
```

Comment:
You don't need the bit shifting here.  The function handles that, actually. 

```suggestion
        set_single_persistent_default_layer(_QWERTY);
```

Reply:
fixed